tests/intermittents_commenter/expected_comment.text

```diff
@@ -5,8 +5,9 @@ This is the #1 most frequent failure this week.
 Repository breakdown:
 * test_treeherder_jobs: 1
 
-Platform breakdown:
+Platform and build breakdown:
 * b2g-emu-jb: 1
```

Comment:
note: we should remove this old reference to b2g, just for clarity

Reply:
So... changing this would cause other tests to break and I'd rather not spend time fixing that right now :) We do need to clean up old test data in general at some point.